src/servestatic/asgi.py

```diff
@@ -47,6 +47,7 @@ async def __call__(self, scope, receive, send):
         wsgi_headers = {
             "HTTP_" + key.decode().upper().replace("-", "_"): value.decode() for key, value in scope["headers"]
         }
+        wsgi_headers["QUERY_STRING"] = scope["query_string"].decode()
```

Comment:
Add a comment here and a better method would be to remove the duplicate value via `pop`

```suggestion
        # The query string header should not begin with `HTTP_`
        wsgi_headers["QUERY_STRING"] = wsgi_headers.pop("HTTP_QUERY_STRING")
```

Reply:
Could you be more specific? From what I can tell, query string isn't a header as it's set on the scope directly under the key `query_string` (see [asgi spec](https://asgi.readthedocs.io/en/latest/specs/www.html#http-connection-scope)).

Reply:
Ah my bad, you're correct.